superduperdb/container/model.py

```diff
@@ -429,6 +435,24 @@ def __post_init__(self):
         else:
             self.to_call = getattr(self.object.artifact, self.predict_method)
 
+    @override
+    def on_load(self) -> None:
+        device_was_found = False
+
+        for device in self.preferred_devices:
+            logging.info(f"Looking for {device}...")
+            try:
+                getattr(self.object.artifact, self.to_method)(device)
```

Comment:
self.to_method 
should be implemented in the future subclasses of the Model class which are 

superduperdb/ext/torch/model.py 
example

You could implement `to_method` attribute of TorchModel class 

```python
class TorchModel:
    def to_method(self, device):
        # something like this
        self.object.artifact.to(device)
```
and this should work

Reply:
Addressed, let me know your thoughts